qp_shotgun/humann2/humann2.py

```diff
@@ -84,50 +125,62 @@ def generate_humann2_analysis_commands(forward_seqs, reverse_seqs, map_file,
             % ', '.join(sn_by_rp.keys()))
 
     cmds = []
-    params = ' '.join(["%s %s" % (k, v) for k, v in viewitems(parameters)])
+
```

Comment:
Note that the output of KneadData is typically going to give us four (uncontaminated) fastq files:
1) R1 reads whose mate also passed
2) R2 reads whose mate also passed
3) R1 reads whose mate didn't pass
4) R2 reads whose mate didn't pass

For Humann2 in the current implementation, read pairing is not considered so these could be concatenated or something. But for some datasets we may need to handle additional cases here.